components/card/card.css

```diff
@@ -0,0 +1,246 @@
+@import url(/cssReset/reset.css);
+@import url(/cssUtilities/utilities.css);
+@import url(/components/avatar/avatar.css);
+@import url(/components/button/button.css);
+@import url(/components/badge/badge.css);
+
+:root {
+  --card-description-color: #808080;
+}
+
+.component-container {
+  display: flex;
+  justify-content: space-evenly;
+  padding: 1rem 0;
+}
+
+/* ||------ Standard Card start */
+
+.card {
+  height: max-content;
+  max-width: 300px;
+  border-radius: var(--large-border-radius);
+  box-shadow: 0 2px 7px rgba(0, 0, 0, 0.2);
+  position: relative;
+}
+
+/* Card Header */
+.card .card-header {
+  display: none;
+}
+
+/* Card image  */
+.card .card-body .card-image img {
+  object-fit: cover;
+  width: 100%;
+  height: 300px;
+  border-top-left-radius: var(--large-border-radius);
+  border-top-right-radius: var(--large-border-radius);
+}
+
+.card.card-small .card-image img {
+  height: 200px;
+}
+
+/* dismiss icon */
+.card .card-body .dismiss-icon {
+  color: var(--card-description-color);
+  display: none;
+}
+/* Card Badge */
+.card .card-body .card-icons {
+  display: none;
+}
+
+.card-icons .card-like-icon .icon:hover {
+  color: red;
+}
+
+/* Card Text Content */
+.card .card-body .card-text-content {
+  margin: 1rem 1.5rem;
+}
+
+.card .card-body .card-text-content .card-description {
```

Comment:
You should avoid nesting too much where possible

Reply:
Okay got it i'll make the changes